boards/gd32307c_eval.json

```diff
@@ -16,7 +16,7 @@
   },
   "debug": {
     "jlink_device": "GD32F307VC",
-    "openocd_target": "gd32f30x",
+    "openocd_target": "gd32f3x0",
```

Comment:
The URL down there links to https://www.gigadevice.com/microcontroller/gd32f303vct6. The `mcu` field says gd32f307vct6. Both of these fall in the "30x" category. Something got confused here?

Reply:
No, that was intentional.  The OpenOCD script is the same for both part families, so until OpenOCD gets updated with scripts for every MCU family this works.

Reply:
Where is that OpenOCD distributation? At https://github.com/GigaDevice-Semiconductor/openocd/tree/xpack/tcl/target there is no such script. 

Also still then, this file has a mismatch in the `mcu` and the `url` fields 
`"mcu": "gd32f307vct6",`
vs
`https://www.gigadevice.com/microcontroller/gd32f303vct6/`

Reply:
OK, so it looks like I may have created those file in .platformio\packages\tool-openocd\scripts\target.  Looks liek I will need to see about contributing them to the OpenOCD project.  For now I will re-test with the STM32 equivalent and verify operation.  And you are correct about the link - the 303 -> 307 change eluded me.

Reply:
Alight, let me know when the commit is there for the merge :)

Reply:
Max I have the board files corrected with the proper OpenOCD target.  I also cleaned up the board files to have the same spacing and format conventions.  Did we decide that the "spl_series" needs to be all capitals or mixed-case?  I hope to have these board files fully corrected with this check-in.

Reply:
SPL series needs to be mixed case for later Arduino core integration where the folders have the verbatim name, see https://github.com/CommunityGD32Cores/ArduinoCore-GD32/tree/main/system. The SPL builder script always lowercases them.

Reply:
>I have the board files corrected with the proper OpenOCD target

Have you? I'm still seeing 

```
"openocd_target": "gd32f10x",
```
in every file. Or did you push these target configs anywhere?

Reply:
Looks good to merge now right?

Reply:
I believe so.  This PR should be good to go.